### DISCUSSION TITLE
https://cucoders.dev/publicaciones/elmoisedecuba/ventajas-de-utilizar-next-js-13-en-tu-proyecto-/

### DISCUSSION THREAD

**Discussion by giscus** [CuCodersCommunity/cucoderscommunity.github.io#69] [Page Comments]
# https://cucoders.dev/publicaciones/elmoisedecuba/ventajas-de-utilizar-next-js-13-en-tu-proyecto-/

Para un buen SEO, es crucial tener etiquetas meta en su aplicación web. Con Next.js 13, ahora puede establecer fácilmente etiquetas meta básicas con la nueva API Head(). Por ejemplo, pue

https://cucoders.dev/publicaciones/elmoisedecuba/ventajas-de-utilizar-next-js-13-en-tu-proyecto-/

**Comment by KeimaSenpai** [comment]
Muy bueno gracias por decir las ventajas. Así veré si lo uso para mis proyectos.
